handlers/config_update.go

```diff
@@ -7,14 +7,38 @@ import (
 	"github.com/rancher/agent/utilities/utils"
 	revents "github.com/rancher/event-subscriber/events"
 	"github.com/rancher/go-rancher/v2"
+	"net/http"
 	"os"
 	"os/exec"
+	"runtime"
 )
 
 type ConfigUpdateHandler struct {
 }
 
 func (h *ConfigUpdateHandler) ConfigUpdate(event *revents.Event, cli *client.RancherClient) error {
+	if runtime.GOOS == "windows" {
+		// if os is windows, do a fake reply and manually touch some urls as @ibuildthecloud suggested
+		items := []string{"host-config", "pyagent", "configscripts"}
```

Comment:
These items should be in the event that is sent.  It will tell you what items to update.